src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../App.css';
+
+const Header = () => {
+    return (
+        <header className="App-header">
+            <h1 className="App-title">React Timeline</h1>
```

Comment:
Just noting you could also use `timelineData.person` to add the person's name to the title.